client/src/layouts/header/searchForm/cities/city/City.jsx

```diff
@@ -1,7 +1,17 @@
 import React from 'react';
+import classes from './city.module.css';
+import nightCloudyWeatherIcon from '../../../../../../public/images/nightCloudyWeatherIcon.png';
 
-function City() {
-  return <div></div>;
+function City({ city }) {
+  return (
+    <div className={classes.container}>
+      <div className={classes.cityWeatherIconAndCity}>
+        <img src={nightCloudyWeatherIcon} className={classes.cityWeatherIcon} />
+        <span>{city.name}</span>
+      </div>
+      <span>{city.temperature}°C</span>
+    </div>
+  );
 }
 
 export default City;
```

Comment:
Please use one style guide for naming folders/components/styles. Why have you use uppercase of first letter in component name and use lowercase of first letter in folder name and in style name?